src/tabs/configuration.html

```diff
@@ -587,17 +587,17 @@
                                     </label>
                                 </div>
                                 <div class="number vbatmonitoring">
-                                    <label> <input type="number" name="mincellvoltage" step="0.1" min="1" max="5" /> <span
+                                    <label> <input type="number" name="mincellvoltage" step="0.01" min="1" max="5" /> <span
```

Comment:
We need to add an API version check to `configuration.js`, and set the step size for these fields to `0.1` if the API is below 1.41. Otherwise users of older versions will attempt to set voltages in 0.01 V resolution and report bugs when they appear rounded after saving / reloading.